src/pdffontparsertype1.cpp

```diff
@@ -1212,11 +1211,10 @@ wxPdfFontParserType1::ReadPFM(wxInputStream& pfmFile)
   // the hyphen to a space.  This actually works in a lot of cases.
   wxString fullName = fontName;
   fullName.Replace(wxT("-"), wxT(" "));
-  wxString familyName = wxEmptyString;

```

Comment:
Indeed this looks fishy. Neither `fullName` nor `familyName` are currently used. Most probably my intention was to set the font data attributes for font family name and full font name. However, somehow I didn't finish the implementation.

Up to now nobody complained about the missing attribute values - most likely, since `.pfm` files are seldom used to load fonts. TrueType or OpenType fonts are usually preferred.